docs/experiments-deepimpact.md

```diff
@@ -0,0 +1,79 @@
+# Anserini: DeepImpact for MS MARCO Passage Ranking
```

Comment:
Add a link to the Anserini docs and say something like - it's just exactly like Anserin, except you can do it from Python.